src/main/java/website/frontrow/board/Player.java

```diff
@@ -140,7 +158,8 @@ public int getLives()
     public void setLives(int l)
     {
     	lives = l;
-    	addToLog("[PLAYER]\t Player's current amount of lives is now: " + lives);
+        observers.forEach(o -> o.livesChanged(this));
```

Comment:
Indenting, line 160 has a tab, line 161 only has spaces.
There's a visible difference, because of this.